client/src/leap/soledad/client/api.py

```diff
@@ -70,6 +71,26 @@
 """
 SOLEDAD_CERT = None
 
+# A whitelist of modules from where to collect plugins dynamically.
+# For the moment restricted to leap namespace, but the idea is that we can pass
+# other "trusted" modules as options to the initialization of soledad.
+PLUGGABLE_LEAP_MODULES = ('mail', 'keymanager')
+
+
+def collect_plugins(interface):
```

Comment:
At some point in the future we might need to use our memoize decorator here as it is run on every sync, and I think if the list of trusted modules is long it might take some time to run. But I don't think it will be necessary now.

Another thing that might be needed in the future is to move this function to `leap.common` adding a `trusted_modules` param. This will make easier for other modules to implement plugin interfaces.